src/app/utils/database-testing.module.ts

```diff
@@ -26,7 +27,8 @@ import { AppModule } from "../app.module";
   ],
 })
 export class DatabaseTestingModule {
-  constructor(pouchDatabase: PouchDatabase) {
+  constructor(pouchDatabase: PouchDatabase, components: ComponentRegistry) {
+    components.allowDuplicates();
```

Comment:
is that really linked with the Database in any way - i.e. should it be here or rather in a different test setup or module?